### PR TITLE
Add teta_cz spider (509 locations)

### PR DESCRIPTION
**Pull request by vfosnar**

### CODE REVIEW

**Comment by vfosnar** [comment]
I'm iterating over all branches and getting their opening hours individually and it looks like the spider times out because of that in the CI. Should I worry about it?

**Comment by Cj-Malone** [comment]
CI only runs for about 1 minuet. It's not an issue, the weekly runs the spider to completion.

**Comment by vfosnar** [comment]
I think the name shouldn't be "cleaned up" as that's the actual name of the branch owner/operator. In maps the brand name "Teta" should be used everywhere anyway.

**Comment by vfosnar** [comment]
Otherwise thx for the code cleanup<3

File: locations/spiders/teta_cz.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                )
+            },
+            callback=self.parse_locations,
+        )
+
+    def parse_locations(self, response, **kwargs):
+        for location in response.json():
+            item = DictParser.parse(location)
+            if item["name"] == "Teta drogerie a lékárny ČR s.r.o.":
+                item["name"] = "Teta"
```
Comment (Cj-Malone) [inline locations/spiders/teta_cz.py:31]:
```suggestion
            item["operator"] = item.pop("name"):
```

I only did `name=Teta drogerie a lékárny ČR s.r.o.`, if the following are better as operator, we should just move them there.

```
name=A - market s.r.o.
name=BATUL s.r.o.
name=DROGERIE A+L s.r.o.
name=DROGERIE JEVÍČKO s.r.o.
name=DROGERIE MARKET Group s.r.o.
name=Drogerie DUHA s.r.o.
name=EMESA s.r.o.
name=FESLOVÁ s.r.o.
name=Hladík Petr
name=Ing. Ngan Bui Van
name=Jiří Dohnal s.r.o.
name=Jiří Watzinger
name=Kolarevič Dušan
name=Martin Pech
name=Milan Hlavatý
name=Novysedláková Štěpánka
name=OneTech, s.r.o.
name=Paroubková Marcela
name=PeMi drogerie s.r.o.
name=Petr Doubek
name=Růžena Prokopová
name=SYNERGIC,s.r.o.
name=Susy s.r.o.
name=TETAX s.r.o.
name=Umlauf Miroslav
name=Šuška Milan
name=Šťastný s.r.o.
```

**Comment by CloCkWeRX** [comment]
### Brand name

Teta drogerie a lékárny ČR s.r.o.

Czech pharmacy

### Wikidata ID

Q20860823
https://www.wikidata.org/wiki/Q20860823
https://www.wikidata.org/wiki/Special:EntityData/Q20860823.json

### Store finder url(s)

Official Url(s): https://www.tetadrogerie.cz
- [x] I have run the below and if anything in generated, added it.
- [ ] I have manually checked for a storefinder
- [ ] This is not a manufacturer listing outlets operating under their own brands / this will not create duplicates
`pipenv run scrapy sf --brand-wikidata=Q20860823 https://www.tetadrogerie.cz`
```
```

**Review by Cj-Malone — APPROVED** [review]
LGTM, thank you.

**Comment by Cj-Malone** [comment]
How did you find out about ATP? What are you planning to use the data for?

**Comment by vfosnar** [comment]
I'm preparing a bot for automatic tag updates (mostly opening hours) from ATP around the Czech Republic.

**Pull request merged**